setup.py

```diff
@@ -136,6 +136,15 @@ def run(self):
 
         SDist.run(self)
 
+        # Print warnings at the end because no one will see warnings before all the normal status
+        # output
+        if os.environ.get('_ANSIBLE_SDIST_FROM_MAKEFILE', False) != 1:
```

Comment:
I believe it's never true, because environ values are strings.
```suggestion
        if os.environ.get('_ANSIBLE_SDIST_FROM_MAKEFILE', False) != '1':
```

Reply:
Hmm... I think you're right but I tested this earlier and it worked.  Let me test again to see why that was.

Reply:
I think I know what happened.  I was testing this but then ran into the changelog issues and never got back to verifying that this worked.